crates/bevy_reflect/src/enums/enum_trait.rs

```diff
@@ -130,6 +130,13 @@ pub trait Enum: Reflect {
     fn variant_path(&self) -> String {
         format!("{}::{}", self.reflect_type_path(), self.variant_name())
     }
+
+    /// Will return `None` if [`TypeInfo`] is not available.
+    ///
+    /// [`TypeInfo`]: crate::TypeInfo
```

Comment:
Docs isn't consistent with the other methods, is there a reason that here we have this second line but in other places not?

Reply:
In doc comments I've seen in Bevy the convention is to prefer the shorter syntax without path qualifiers.
But doing so requires the type to be in scope.
This file did not have `TypeInfo` in scope, the others did.
Rust allows bringing types into scope via the syntax you see on the last line.